internal/seleniumtest/seleniumtest.go

```diff
@@ -1098,6 +1100,50 @@ func testGetAttributeNotFound(t *testing.T, c Config) {
 	}
 }
 
+func testGetProperty(t *testing.T, c Config) {
+	wd := newRemote(t, newTestCapabilities(t, c), c)
+	defer quitRemote(t, wd)
+
+	if err := wd.Get(c.ServerURL); err != nil {
+		t.Fatalf("wd.Get(%q) returned error: %v", c.ServerURL, err)
+	}
+	input, err := wd.FindElement(selenium.ByName, "q")
+	if err != nil {
+		t.Fatalf("Can't find element: %v", err)
+	}
+	const query = "golang"
+	if err := input.SendKeys(query); err != nil {
+		t.Fatalf("Can't send keys: %v", err)
+	}
+
+	time.Sleep(500 * time.Millisecond)
```

Comment:
Add a comment as to why this sleep is needed.

Reply:
I removed this line.
I copied this from the [testSendKeys function](https://github.com/tebeka/selenium/blob/master/internal/seleniumtest/seleniumtest.go#L584) and I thought it was needed.